scripts/check_regression.sh

```diff
@@ -36,7 +37,19 @@ compare_stat_p90() {
 
 calculate_threshold() {
     local past_value="$1"
-    awk -v past="$past_value" 'BEGIN { print past * 1.1 }'
+    awk -v past="$past_value" 'BEGIN { print past * 1.5 }'
+
+}
+
+calculate_p90_after_skip() {
```

Comment:
nit: can you add a comment that this calculates the p90 ignoring the first result because we generally see an outlier in the first result?

Reply:
sure